### DISCUSSION TITLE
Reset 'Various Artists' when item is updated with a proper tag

### DISCUSSION THREAD

**Discussion by vingerha** [music-assistant/support#2644] [Feature requests and ideas]
When scanning items, if the tag 'Album Artist' is missing, MA sets that value to 'Various Artists'. In my large local music collection on filesystem this leads to hundreds of such repair-tagging. Setting the tag properly on the items is a tedious task so I do this gradually. 
However, the scan/sync does not picks up on those changes, they remain under 'Various Artists' and the solution suggested is to remove the items and rescan...also this is a tedious action. 
My request: for items that have an Album Artist 'Various Artists' to perform an extra check if this changed. 
As there are many situations where these never change, the scanning should not run pointlessly each/every time. Options could be:
- special menu item to do this
- weekly scan
- bulk-remove based on an Album Artist

**Comment by OzGav** [comment]
I just tried this and this now seems to work as you expected. I added a track after having removed the album artist and associated release artist MB ID. I scanned it in and it was assigned the various artists album artist. I then retagged the file in place. On the next sync the old entry was removed and new one created.

**Comment by vingerha** [comment]
This is an oldie and if you say t works (now), then I am happy to close it. Thanks :)
